scripts/synthetix/facilitate_trade.js

```diff
@@ -50,6 +51,20 @@ const gasStationURL = {
   4: "https://safe-relay.rinkeby.gnosis.io/api/v1/gas-station/",
 }
 
+const estimationURLPrexix = {
+  1: "https://dex-price-estimator.gnosis.io//api/v1/",
+  4: "https://dex-price-estimator.rinkeby.gnosis.io//api/v1/",
+}
+
+const estimatePrice = async function (buyTokenId, sellTokenId, amount, networkId) {
+  const searchCriteria = `markets/${buyTokenId}-${sellTokenId}/estimated-buy-amount/${amount}?atoms=true`
+  const estimationData = await (await fetch(estimationURLPrexix[networkId] + searchCriteria)).json()
+
+  return estimationData.buyAmountInBase / estimationData.sellAmountInQuote
```

Comment:
I think the expected output when specifiec sell amount cant be filled is precisely what we are seeing in the case of rinkeby test plan (above). i.e. infinity and zero.

You can try these out yourself buy adding zeros to the following query:


https://dex-price-estimator.gnosis.io//api/v1/markets/8-9/estimated-buy-amount/10000000000000000000?atoms=true

> If so, it might make sense to check that estimationData.sellAmountInQuote === amount for our purposes.

In any case, yes we can throw in this assertion but I believe it will always be true.

Reply:
I see! Then I wouldn't change it, at most a comment explaining this.